src/main/java/com/rtb/projectmanagementtool/project/DeleteProjectServlet.java

```diff
@@ -0,0 +1,43 @@
+/** Servlet responsible for deleting a project */
+package com.rtb.projectmanagementtool.project;
+
+import com.google.appengine.api.datastore.*;
+import com.rtb.projectmanagementtool.task.*;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/delete-project")
+public class DeleteProjectServlet extends HttpServlet {
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
```

Comment:
GET shouldn't forward to POST.  They're sematically different and should handle different types of requests.